src/lookup-table/models/LookupTable.model.ts

```diff
@@ -207,11 +215,20 @@ const LookupTableModel = () => {
 
   const updateStructureFieldMaps = async (
     tableName: string,
-    fieldMaps: FieldMapType[]
+    name: string,
+    fieldMaps: FieldMapType[],
+    dataViewCode: string
   ): Promise<boolean> => {
-    const text = `UPDATE data_table SET field_map = $1 WHERE table_name = $2`
+    const text = `UPDATE data_table SET
+      display_name = $1,
```

Comment:
Also allows for the "display_name" to be updated as well as the code (reflected in front-end form)